src/ds/hashes.md

```diff
@@ -0,0 +1,47 @@
+Lists, sets, and sorted sets are great for many use cases, but the hash data type is on a higher level. Redis hashes are maps (sequences of field-value pairs) and are used to represent objects in Redis. Consider the following example that shows how to create a hash key using the `HSET` command.
```

Comment:
- Consider rewording "is on a higher level". It is wordy and 'm not sure what it means.
- "used to represent objects in Redis" can be misleading because objects can mean different things. Consider the following definition from https://redis.io/docs/data-types/hashes/: "Redis hashes are record types structured as collections of field-value pairs. "